charts/opensearch-dashboards/values.yaml

```diff
@@ -146,11 +146,12 @@ resources:
     memory: "512M"
 
 autoscaling:
-  enabled: false
+  # This requires metrics server to be installed, to install use kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+  # See https://github.com/kubernetes-sigs/metrics-server
+  enabled: true
```

Comment:
IMO Lets not keep it enabled by default. For resources we can keep it silenced, this should not blast someone's cluster. Also this will fail things if metric server is not installed in the setup.

Reply:
Fixed in latest commit.